ghost/core/core/server/services/mega/feedback-buttons.js

```diff
@@ -0,0 +1,69 @@
+const {Color} = require('@tryghost/color-utils');
+
+const templateStrings = {
+    like: '%{feedback_button_like}%',
+    dislike: '%{feedback_button_dislike}%'
+};
+
+const generateLinks = (postId, uuid, html) => {
+    const audienceFeedback = require('../audience-feedback');
```

Comment:
Why is this `require`d here as opposed to the top of the file?

Reply:
I think, it's just a result of my experiments. All services started in `boot` file, moved it to the top :)